src/pages/VaultsPage/CreateVaultModal.tsx

```diff
@@ -42,15 +43,18 @@ export const CreateVaultModal = ({
   open: boolean;
   onClose: () => void;
 }) => {
-  const [asset, setAsset] = useState<string>();
+  const [asset, setAsset] = useState<string>('');
 
   const classes = useStyles();
 
   const selectedCircle = useSelectedCircle();
 
   const history = useHistory();
 
-  const routeChange = () => {
+  // const { _createApeVault } = useVault();
```

Comment:
I needed to modify this in master. I removed the dependencies on unused data hooks, so it wouldn't conflict with my branch.

I also updated it a bit to use the `AdminVaultForm.FormController`
All of these modals are simple, so there is probably a way to do the forms without needing to have 5 times forms.

These were based on the `EditUserForm`, `EditProfileForm` is a simpler example.

### I believe these should mostly be easy to merge over
https://github.com/coordinape/coordinape/pull/291